strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/services/ArtifactManagementServiceImplTest.java

```diff
@@ -453,36 +413,50 @@ public void testArtifactResolutionFromGroup(TestInfo testInfo)
         }
     }
 
+    @ExtendWith({ RepositoryManagementTestExecutionListener.class, ArtifactManagementTestExecutionListener.class })
     @Test
-    public void testForceDelete(TestInfo testInfo)
+    public void testForceDelete(@TestRepository(storage = STORAGE0,
+                                                repository = REPO_PREFIX8+"-testForceDelete",
```

Comment:
Cosmetic: In our project, we leave spaces around operators such as the `+` sign.

Reply:
Sure.